docs/intro.rst

```diff
@@ -1,236 +0,0 @@
-.. _schemaregistry_intro:
-
-Introduction
-============
```

Comment:
@joel-hamill I'm curious why this page was deleted. 

Are the curl examples considered duplicate information with the API Reference page?

Reply:
@cricket007 this content was moved to https://docs.confluent.io/current/schema-registry/docs/installation.html, but it looks like part of it was accidentally left off. adding back in a follow-on PR.

Reply:
@cricket007 this information is now here: https://docs.confluent.io/current/schema-registry/docs/using.html#common-sr-usage-examples

Reply:
Thanks @joel-hamill ! 
I made some additions to that page in #876 . Looking forward to a review.